src/ert/_c_wrappers/enkf/config/gen_kw_config.py

```diff
@@ -82,7 +84,7 @@ def __init__(
         self._set_template_file(template_file)
         self.__str__ = self.__repr__
 
-    def getTemplateFile(self):
+    def getTemplateFile(self) -> os.PathLike:
         path = self._get_template_file()
```

Comment:
`os.PathLike` is hella weird. In Python, that is the name of the type, but for whatever reason, in mypy context, it's a generic that takes in the type of the underlying string. If this is failing, do `os.PathLike[str]` (you need to have `from __future__ import annotations` for this, which you do in this file)

Reply:
Thanks, I added [str]